src/main/resources/edu/cornell/kfs/gl/document/workflow/GeneralLedgerTransferDocument.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This is the KualiCo-delivered General Ledger Transfer Document workflow XML,
+    which has been separated out into its own separate file for ingestion convenience.
+ -->
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+    <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+        <documentType>
+            <name>
+                GLT
+            </name>
+            <parent>
+                FP
+            </parent>
+            <label>
+                General Ledger Transfer
+            </label>
+            <docHandler>
+                ${kfs.url}/financialGeneralLedgerTransfer.do?methodToCall=docHandler
+            </docHandler>
+            <helpDefinitionURL>
+                default.htm?turl=WordDocuments%2Fgeneralledgertransfer.htm
+            </helpDefinitionURL>
+            <active>
+                true
+            </active>
+            <routingVersion>
+                2
```

Comment:
since this is a new document type, shouldn't the version be 1?

Reply:
The "routingVersion" doesn't refer to a BO version number; it refers to the KEW doc type routing setup. Version 1 specifies an archaic "route level" routing scheme, whereas version 2 represents the route node routing scheme that is used today.

Reply:
ah, right